owtf/webapp/src/containers/LoginPage/index.js

```diff
@@ -1,19 +1,95 @@
 /*
  * LoginPage.
+ * Manages Login and handles sending the login request and setting the login token
  */
-import React from 'react';
-import UnderconstructionPage from 'components/UnderconstructionPage';
 
-export default class LoginPage extends React.Component {
-  // Since state and props are static,
-  // there's no need to re-render this component
-  shouldComponentUpdate() {
-    return false;
+import React from "react";
+import {
+  Pane,
+  Heading,
+  Button,
+  Text,
+  Link,
+  TextInputField
+} from "evergreen-ui";
+import {
+  GoogleLoginButton,
+  GithubLoginButton,
+  TwitterLoginButton
+} from "react-social-login-buttons";
+const OAuthBtnStyle = { width: "60%", marginLeft: "20%" };
+
+export class LoginPage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      email: "",
+      password: ""
+    };
   }
 
   render() {
     return (
-      <UnderconstructionPage />
+      <Pane marginY={90}>
+        <Pane justifyContent="center" width="35%" elevation={1} margin="auto">
+          <Heading
+            size="700"
+            textAlign="center"
+            marginBottom={20}
+            paddingTop={30}
+          >
+            Login
+          </Heading>
+          <TextInputField
+            placeholder="Email"
+            width="60%"
+            marginLeft="20%"
+            marginBottom={20}
+            value={this.state.email}
+            onChange={e => this.setState({ email: e.target.value })}
+          />
+          <TextInputField
+            placeholder="Password"
+            width="60%"
+            marginLeft="20%"
+            marginBottom={20}
+            value={this.state.password}
+            onChange={e => this.setState({ password: e.target.value })}
+          />
+          <Text width="60%" marginLeft="20%" marginBottom={20}>
+            <Link href="/forgot-password">Forgot Password?</Link>
+          </Text>
+          <Button
+            width="20%"
+            marginLeft="40%"
+            marginBottom={10}
+            marginTop={20}
+            justifyContent="center"
+            appearance="primary"
+            intent="none"
+          >
+            LOGIN
+          </Button>
```

Comment:
alignment seems off while testing on my monitor
![Screenshot from 2021-06-13 23-34-30](https://user-images.githubusercontent.com/31919606/121817659-4fc3d580-cca0-11eb-8e93-881205ef7100.png)

Reply:
Corrected it. The problem was mostly because of the use of Text element provided by "evergreen-ui" which is defaulted to <span> HTML element. Changed it with Paragraph element which translates to <p> element in HTML and since <p> is a block element, it will cover the entire width of the provided space.